src/wp-includes/blocks.php

```diff
@@ -1344,6 +1344,60 @@ function traverse_and_serialize_block( $block, $pre_callback = null, $post_callb
 	);
 }
 
+function replace_pattern_blocks( $blocks, &$inner_content = null ) {
```

Comment:
The function documentation is missing.

Reply:
The function name is also rather generic